database/migrations/sqls/20240816190503-createMessageTable-up.sql

```diff
@@ -4,6 +4,7 @@ CREATE TABLE queue.message (
     id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
     channel text,
     data jsonb,
+    ack jsonb NOT NULL DEFAULT '{}'::jsonb,
```

Comment:
This is fine to modify your sql here, but in the future, we need add new `up` to alert table and add column, this is correct way to change db with path of software evolution.

Reply:
Understood. I already ran the change on the database which is why I modified the existing SQL script.